### PR TITLE
pandasdmx 1.0.0 recipe

### PR DESCRIPTION
**Pull request by dr-leo**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pandasdmx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+    {% set name = "pandaSDMX" %}
```
Comment (synapticarbors) [inline recipes/pandasdmx/meta.yaml:1]: fix indentation

File: recipes/pandasdmx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 799d53f8c8a49321fc92d8bf0f6bf7101b0ffe14c7f2768aef33776b7eae48f6
+
+build:
+  number: 0
+  skip: true  # [py<37]
```
Comment (synapticarbors) [inline recipes/pandasdmx/meta.yaml:14]: I think you can remove this `skip` statement, add `noarch: python` (https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) and change `python` to `python >=3.7` below under the `host` and `run` requirements sections.

File: recipes/pandasdmx/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+build:
+  number: 0
+  skip: true  # [py<37]  
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv" 
+
+requirements:
+  host:
+    - python
+    - pip
```
Comment (synapticarbors) [inline recipes/pandasdmx/meta.yaml:20]: it looks like you'll need `flit` under host

File: recipes/pandasdmx/meta.yaml (hunk 2 above)
Comment (dr-leo) [reply recipes/pandasdmx/meta.yaml]: Thanks. Works. Anything else to do? Next steps?

**Comment by synapticarbors** [comment]
@dr-leo -- Thank you for your contribution to conda-forge. Once the feedstock for the package is created, if you run into problems, don't hesitate to get in touch.

**Pull request merged**